service-mesh/istio/ecommerce-app.yaml

```diff
@@ -43,6 +43,8 @@ spec:
         ports:
         - containerPort: 5000
         env:
+        - name: SIGNALFX_SERVICE_NAME
+          value: "api"
```

Comment:
seems like these could be in the istio branch

Reply:
This was showing up because of a messed up rebase. Fixed now